README.md

```diff
@@ -1,45 +1,174 @@
-[![Coverage Status](https://coveralls.io/repos/github/serverless/serverless/badge.svg?branch=v1.0)](https://coveralls.io/github/serverless/serverless?branch=dev)
-[![gitter](https://img.shields.io/gitter/room/serverless/serverless.svg)](https://gitter.im/serverless/serverless)
-[![dependencies](https://img.shields.io/david/serverless/serverless.svg)](https://www.npmjs.com/package/serverless)
-[![license](https://img.shields.io/npm/l/serverless.svg)](https://www.npmjs.com/package/serverless)
+[Website](http://www.serverless.com) • [Email Updates](http://eepurl.com/b8dv4P) • [Gitter (1,000+)](https://gitter.im/serverless/serverless) • [Forum](http://forum.serverless.com) • [Meetups (7+)](https://github.com/serverless-meetups/main) • [Twitter](https://twitter.com/goserverless) • [Facebook](https://www.facebook.com/serverless) • [Contact Us](mailto:team@serverless.com)
+
+**The Serverless Framework** – Build applications comprised of microservices that run in response to events, auto-scale for you, and only charge you when they run.  This lowers the total cost of maintaining your apps, enabling you to build more logic, faster.
+
+The Framework uses new event-driven compute services, like AWS Lambda, Google CloudFunctions, and more.  It's a command line tool, providing scaffolding, workflow automation and best practices for developing and deploying your serverless architecture. It's also completely extensible via plugins.
+
+Serverless is an MIT open-source project, actively maintained by a full-time, venture-backed team.  Get started quickly by [watching the video guide here](https://www.youtube.com/watch?v=jfDSc4qq440).
+
+Enjoy! - Serverless, Inc.
+
+## Links
+
+* [Quick Start](#quick-start)
+* [Features](#features)
+* [Documentation v.1](#documentation) / [v.0](http://docs.serverless.com/)
+* [Road Map](https://github.com/serverless/serverless/milestones)
+* [Contributing](#contributing)
+* [Community](#community)
+* [Changelog](https://github.com/serverless/serverless/releases)
+
+## <a name="quick-start"></a>Quick Start
+
+[Watch the video guide here](https://www.youtube.com/watch?v=jfDSc4qq440) or follow the steps below to create and deploy your first serverless microservice in minutes.
 
-The Serverless Framework is the application framework for building web, mobile and IoT applications on serverless, event-driven compute services, like [AWS Lambda](https://aws.amazon.com/lambda/), [Google CloudFunctions](https://cloud.google.com/functions/), [Azure Functions](https://azure.microsoft.com/en-us/services/functions/) and more.
+| **Step** | **Command** |**Description**|
+|---|-------|------|
+|  1.  | `npm install -g serverless@beta` | Install Serverless CLI  |
+|  2.  | `mkdir my-first-service && cd my-first-service` | Create the project directory |
+|  3.  | [Input your provider credentials](/docs/guide/provider-account-setup.md#amazon-web-services) | Connect Serverless with your provider |
```

Comment:
Don't want to send people to the AWS dashboard first.  Want them to install the tool first.  Changed the language for better clarity.  Let's leave it for now.